File: 0392-is-subsequence/0392-is-subsequence.py
class Solution(object):
    def isSubsequence(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s) > len(t):return False
        if len(s) == 0:return True
        subseq = 0
        for i in range(0,len(t)):
            if subseq <= len(s) -1:
                print(s[subseq])
                if s[subseq]==t[i]:
                    subseq+=1
        return subseq == len(s)

File: 0392-is-subsequence/0392-is-subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        int st = 0;
        int tt = 0;
        while(st < s.length() && tt < t.length()){
            if(s.charAt(st) == t.charAt(tt)){
                st++;
            }
            tt++;
        }
        return st == s.length();
    }
}